### PR TITLE
Ready for Review 

### PR DESCRIPTION
**Pull request by ErinBuckley**
Klynton - this is the complete first draft of the second course - Validating your Puppet Code. You can find the Captivate project, with computer generated voice, on Box: 
/Users/erinbuckley/Box Sync/Education/LMS_OnlineCourses/CourseDevelopment/01-Puppet/TestingBundle

### CODE REVIEW

**Comment by klynton** [comment]
@ErinBuckley Hm, so I don't actually have Captivate. I'll have to get that on Friday.

**Comment by ErinBuckley** [comment]
I published the project so you can view the html or the swf

File: courses/01-puppet/01_2xxx_code_test.md [hunk 1]
```diff
@@ -17,28 +15,109 @@ At the end of this course you will be able to:
...
+And remember, if you want to use modules written by others users, run the `puppet module install` command.
+
+For more information about writing Puppet modules, you can refer to our online documentation at docs dot puppetlabs dot com., and the Puppet Forge, a repository of modules written by our community.
 
-### slide
+### slide Style Guide
 
-### slide 
-"Before you can confirm that your Puppet module can produce the intended result, you need to complete the most basic level of testing - code validation. 
+So, you have written a great module starting with the Puppet module tool. Next you can use the Puppet Language Style Guide to ensure that your modules work as expected and adhere to established standards. You can find the style guide on the Puppet Labs website.
```
Comment (klynton) [inline courses/01-puppet/01_2xxx_code_test.md]: The style guide really doesn't help your modules to "work", just the adhere to established standards thing.

Comment (ErinBuckley) [inline courses/01-puppet/01_2xxx_code_test.md]: OK - i'll change that.

**Pull request merged**